changelog.d/2845.misc.rst

```diff
@@ -0,0 +1,3 @@
+Added :func:`bank_prune` to :module:`redbot.core.bank`
+
+- :func:`bank_prune` can be used to delete a users bank account or remove all users who are not in the guild (bank is local) or visible to the bot (bank is global)
```

Comment:
This sentence has too many `or`s that it isn't clear what you are trying to say. The situation is `A or (B or C)`, however that isn't very clear from the wording.